code/frameworks/experimental-nextjs-vite/src/plugin/index.ts

```diff
@@ -0,0 +1,3 @@
+import vitePluginStorybookNextJs from 'vite-plugin-storybook-nextjs';
```

Comment:
I assume in the future we also want to move the contents of the package into the monorepo?